examples/gen-certs.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -ex
+openssl genrsa -out rootca.key 4096
+bash -c $(dirname $0)/rootca.exp
+openssl genrsa -out cert.key 2048
+bash -c $(dirname $0)/csr.exp
+openssl x509 -req -days 1000 -in cert.csr -CA rootca.crt -CAkey rootca.key -CAcreateserial -out cert.crt -sha1
```

Comment:
The above 3 command can be simplified as:
```
openssl req -new -batch -nodes -sha256 -keyout cert.key -out cert.csr -subj '/C=GB/CN=quic.tech'
openssl x509 -req -days 10000 -in cert.csr -CA rootca.crt -CAkey rootca.key -CAcreateserial -out cert.crt
```
So we don't need the `expect` script here either.

Reply:
Done.